content/en/learn.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Learn
+title: Documentation
 sidebar: false
 ---
 
-To learn how to use NumPy, you could begin with the resources listed under the **'Beginners'** section. Once you gain a broad view, you could try the more involved tutorials and resources listed under the **'Advanced'** section.
+<p style="font-size: 140%;">The official NumPy documentation lives at <a href="https://numpy.org/doc">https://numpy.org/doc.</a></p>
```

Comment:
The larger font doesn't look so good style-wise. It's the first sentence, people will read it either way, so let's do normal font here.

Reply:
Also the `<a href=...` isn't needed; Hugo recognizes links without special markup.

Reply:
You would *think* the first line has prominence.  I'm concerned that because there's **so** much other material people scanning the page won't see it (think of how we look at a new page -- we scan it quickly before reading it).

Is **bold** a reasonable compromise?

Reply:
that does look better. bold italics or simple italics work too. I went with bold

Reply:
Looking at the page I'm feeling better about how the Learn/Documentation issue is turning out. People who click Learn get a chance to re-route to the official docs, without the frustration of having to mentally backtrack and try another nav link.

We can use a symmetric link on the official side pointing to the external docs (or may already have another page of them), but that's less urgent.

Reply:
Just realized: That "external" page includes NumPy User Guide (last entry in Tutorials) -- should be removed.

Reply:
Also -- I'd urge that the hyperlink *not* say 'here'!  It's too easy to not notice the link and misread "here" as meaning "this page".  

Instead of `here`, the URL should be spelled out: https://numpy.org/doc/stable

It should explicitly say https://numpy.org/doc/stable so that nobody thinks "here" me